lib/tool_belt/cherry_picker.rb

```diff
@@ -27,7 +27,7 @@ def find_cherry_picks(project, release, repo_names)
         commits = issue['changesets']
 
         commits.each do |commit|
-          if !commit['comments'].start_with?('Merge pull request') && !@release_environment.commit_in_release_branch?(repo_names, commit['comments'])
+          if commit['comments'].downcase.start_with?('fixes', 'refs') && !@release_environment.commit_in_release_branch?(repo_names, commit['comments'])
```

Comment:
It should be -- and the only way it should make it into Redmine, however, I don't see a reason to limit this here currently. The previous intent was just to ensure ugly 'Merge pull request' commits got ignored.